modules/ui/sections/map_daterange.js

```diff
@@ -0,0 +1,139 @@
+import { t } from '../../core/localizer';
+import { uiSection } from '../section';
+import { utilQsString, utilStringQs, utilNormalizeDateString } from '../../util';
+
+const DEFAULT_MIN_DATE = '-4000-01-01';
+const DEFAULT_MAX_DATE = (new Date()).getFullYear() + '-12-31';
+
+const INPUT_STYLES = [
+    { name: 'width', value: '125px' },
+    { name: 'text-align', value: 'center' },
+];
+const LABEL_STYLES = [
+    { name: 'font-weight', value: 'bold' },
+    { name: 'display', value: 'inline-block' },
+    { name: 'width', value: '75px' },
+];
+
+export function uiSectionDateRange(context) {
+    // despite appearing as a separate panel, Map Features does the real filtering
+    // see applyDateRange() in this panel, where the dateRange value is set
+    // see modules/renderer/features.js checkDateFilter() which applies the filters
+    // see modules/renderer/features.js update() which updates URL params
+
+    const section = uiSection('date_ranges', context)
+        .label(() => t.append('date_ranges.title'))
+        .disclosureContent(renderDisclosureContent)
+        .expandedByDefault(false);
+
+    function renderDisclosureContent(selection) {
+        const container = selection.selectAll('.date_ranges-container').data([0]);
+
+        // start date label & input
+        const mindate_label = container.enter()
+            .append('label')
+            .html(t('date_ranges.start_date.description'))
+            .merge(container);
+        const mindate_input = container.enter()
+            .append('input')
+            .attr('type', 'text')
+            .attr('value', DEFAULT_MIN_DATE)
+            .attr('title', () => t.append('date_ranges.start_date.tooltip'))
+            .attr('placeholder', () => t.append('date_ranges.start_date.placeholder'))
```

Comment:
The tooltip is showing up as a stringified function:

<img width="468" alt="tooltip" src="https://user-images.githubusercontent.com/1231218/194676227-3b11e5fd-5be1-4271-9b42-7c4bbb1a6c13.png">

I was mistaken in https://github.com/OpenHistoricalMap/iD/pull/142#discussion_r986157642 – `t.append()` needs to be called in situations when `t()` used to be passed directly into `html()`, but for an attribute that can’t contain HTML, `t()` is safe.